staging/src/k8s.io/csi-translation-lib/plugins/azure_file.go

```diff
@@ -163,31 +163,48 @@ func (t *azureFileCSITranslator) TranslateCSIPVToInTree(pv *v1.PersistentVolume)
 		ReadOnly: csiSource.ReadOnly,
 	}
 
+	for k, v := range csiSource.VolumeAttributes {
+		switch strings.ToLower(k) {
+		case shareNameField:
```

Comment:
Similar question for shareName. Will it always be identifiable from the volumeHandle?

It would be simpler to eliminate duplicate fields. Then we don't need to think about cases like if the 2 fields have different values.

Reply:
there is also cases there `volumeHandle` is arbitrary string, user input shareName by storage class or PV parameters, we also supports that. If no such input, will parse from volumeHandle

Reply:
Can we consolidate on the volumeHandle format? For example, if the shareName is a storageclass parameter, then the provisioned PV object will have the shareName in the volumeHandle and not the attributes. And for a preprovisioned PV case, the share name has to be provided in the volumeHandle?

Reply:
https://github.com/kubernetes-sigs/azurefile-csi-driver/issues/536